miden-lib/asm/miden/kernels/tx/tx.masm

```diff
@@ -174,84 +174,84 @@ end
 #! position. Returns the pointer to the note the asset was stored at.
 #! Panics if the combined quantity exceeds the maximum for fungible assets (~overflow).
 #!
-#! Inputs: [ASSET, note_ptr, num_of_assets]
+#! Inputs: [ASSET, note_ptr, num_of_assets, note_idx]
```

Comment:
Is `note_idx` used anywhere in here? If not, it is better to remote it.

Reply:
ah the comment is wrong, one second

Reply:
You are right, I just need to store it at the end of the stack. 

Alternatively, I could store it in memory in `export.add_asset_to_note`

Reply:
What I'm trying to say is that if this is not used in the procedure body, then it doesn't have to be annotated as an `input`. The same way a function like this:

```rust
fn func(a: u64, b: u64, c: u64) -> u64 { a + b }
```

Doesn't need the argument `c`, and we would remove it from there. I think here too we should remove `note_idx` from the `Inputs` since it is not used.

IOW, I'm suggesting only removing the mentions to `note_idx` from the comments, and keep the assembly as it is.